src/main/java/com/thealgorithms/maths/LongDivision.java

```diff
@@ -39,14 +39,7 @@ public static int divide(int dividend, int divisor) {
         for (int i = 0; i < dividendString.length(); i++) {
             String partV1 = remainder + "" + dividendString.substring(lastIndex, i + 1);
             long part1 = Long.parseLong(partV1);
-            if (part1 > newDivisor1) {
-                int quotient = 0;
-                while (part1 >= newDivisor1) {
-                    part1 = part1 - newDivisor1;
-                    quotient++;
-                }
-                answer.append(quotient);
-            } else if (part1 == newDivisor1) {
+            if (part1 >= newDivisor1) {
```

Comment:
Why did you remove it? Maybe it wasn't covered by tests? Because it seems to be inefficient but requires code. 

Perhaps this could be written more efficiently, like this:

```
    int quotient = (int) (part1 / newDivisor1);
    part1 = part1 % newDivisor1;
    answer.append(quotient);
```